Playbook.yaml

```diff
@@ -0,0 +1,84 @@
+- name:  App - Java with Maven - Ubuntu Linux
+  hosts: servers
+  become: yes
+  remote_user: user
+  gather_facts: no
+  
+  vars:
+    maven_major_version: 3
+    maven_version: 3.8.6
+  
+  tasks:
+    - name: Install Default Java JRE/JDK Pkg
+      apt:
+        pkg:
+        - default-jre
```

Comment:
what java version is this? Java 8?11? It is a best practice to fix specific package versions, especially when it is the run time of the apps, like java, node etc

Reply:
It's Java 11. Suggestion has been applied.